src/Workspaces/CSharp/Portable/Simplification/CSharpSimplificationService.Expander.cs

```diff
@@ -576,6 +576,26 @@ private ExpressionSyntax VisitSimpleName(SimpleNameSyntax rewrittenSimpleName, S
                             return replacement;
                         }
 
+                        if (replacement.IsKind(SyntaxKind.TupleType))
+                        {
+                            replacement = replacement.WithAdditionalAnnotations(identifier.GetAnnotations());
+
+                            if (_annotationForReplacedAliasIdentifier != null)
+                            {
+                                replacement = replacement.WithAdditionalAnnotations(_annotationForReplacedAliasIdentifier);
+                            }
+
+                            var aliasAnnotationInfo = AliasAnnotation.Create(aliasInfo.Name);
+
+                            replacement = replacement.WithAdditionalAnnotations(aliasAnnotationInfo);
+
+                            replacement = newNode.CopyAnnotationsTo(replacement);
+
+                            replacement = AppendElasticTriviaIfNecessary(replacement, originalSimpleName);
+
+                            return replacement;
+                        }
```

Comment:
i basically hate this entire method.  completely undocumented, and so much duplication across things.   it's not your fault, but i go by the "you're the last person to touch it" maxim.  So you get to make it better :)

Reply:
That's fair :D

Reply:
Refactored the code just around the part I touched :)